### DISCUSSION TITLE
[Base] Suggesting addition of hut information to Base/Infrastructure theme in OverTourMaps

### DISCUSSION THREAD

**Discussion by forwalk-org** [OvertureMaps/data#248] [Discussion]
### Category Feedback

I would like to suggest adding information about huts (basic hut/alpine hut/wilderness hut) to the Base/Infrastructure theme in OverTourMaps. This information would be incredibly valuable for creating thematic hiking maps.

Some sample data from osm https://overpass-turbo.eu/s/1U5G

**Comment by danabauer** [comment]
Thanks for this suggestion, @forwalk-org. We will consider adding huts in a future release.
